tamr_client/categorization/_categorization.py

```diff
@@ -0,0 +1,87 @@
+"""
+Tamr - Categorization
+See https://docs.tamr.com/docs/overall-workflow-classification
+
+The terminology used here is consistent with Tamr UI terminology
+
+Asynchronous versions of each function can be found with the suffix `_async` and may be of
+interest to power users
+"""
+from tamr_client import operation
+from tamr_client._types import CategorizationProject, Dataset, Operation, Session
+from tamr_client.dataset import _dataset, unified
+
+
+def manual_labels(session: Session, project: CategorizationProject) -> Dataset:
+    """Get manual labels from a Categorization project.
+
+    Args:
+        project: Tamr project containing labels
+
+    Returns:
+        Dataset containing manual labels
+
+    Raises:
+        dataset.NotFound: If no dataset could be found at the specified URL
+        dataset.Ambiguous: If multiple targets match dataset name
+    """
+    unified_dataset = unified.from_project(session=session, project=project)
+    labels_dataset_name = unified_dataset.name + "_manual_categorizations"
+    return _dataset.by_name(
+        session=session, instance=project.url.instance, name=labels_dataset_name
+    )
+
+
+def update_unified_dataset(
+    session: Session, project: CategorizationProject
+) -> Operation:
+    """Apply changes to the unified dataset and wait for the operation to complete
+
+    Args:
+        project: Tamr Categorization project
+    """
+    op = _update_unified_dataset_async(session, project)
+    return operation.wait(session, op)
```

Comment:
I think `_update_unified_dataset_async` is not worth it since this function can be written as:

```suggestion
def update_unified_dataset(
    session: Session, project: CategorizationProject
) -> Operation:
    """Apply changes to the unified dataset and wait for the operation to complete

    Args:
        project: Tamr Categorization project
    """
    unified_dataset = unified.from_project(session, project)
    op = unified._apply_changes_async(session, unified_dataset)
    return operation.wait(session, op)
```

...just inlining the helper function. The main intent for the `*_async` functions is to provide an abstraction over `requests` in terms of Tamr concepts, which `unified._apply_changes_async` already does.